frequently-encountered-problems.html

```diff
@@ -248,6 +248,25 @@ <h2>TLS 1.0 compatibility with Node 12 and newer</code></h2>
         </p>
     </dd>
     <br />
+    <dt>
+        <h2>How Tedious handles Unicode datatype conversions when using parameters</code></h2>
+    </dt>
+    <dd>
+        <p>
+            Since all JavaScript strings are Unicode, when using a non-Unicode datatype (such as VarChar), then Tedious will do the conversion 
+            from a Unicode string to a non-Unicode string then send the data to the server. 
+            The conversion may not be what the SQL server expects and cause inaccurate results.
```

Comment:
Maybe we should describe that we're using `iconv-lite` to perform this conversion? The conversion might not match the way SQL Server would do the conversion in some edge cases (like characters that can't be translated to the target encoding).

Reply:
Mentioned iconv-lite and added the additional info on the conversion